draft-scenarios/coops/close_to_enemies.tex

```diff
@@ -0,0 +1,222 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Cooperative Scenario}{Close to enemies}{\images/ballistics.png}
+
+\begin{multicols}{2}
+
+\textbf{Author:} Kelghor
+
+\textbf{Version:} 1.6
+
+\textbf{Source:} \href{https://discord.com/channels/740870068178649108/1313651865648369725}{Archon Studio Discord}
+
+\textit{``Sir! The raiders are at our gates!''}
+
+\textit{``Sound the warhorn! We'll drive them back as always!'' you command, and go to defend the city. Just as you have done many times before. Such is the harsh life of the border clans, where wars and skirmishes with neighboring rivals never truly end. Fortunately, your alliance remains strong, and you know your back is covered.}
+
+\textit{The time has come to end these long-standing wars and crush your enemies like a bug once and for all!}
+% Translator info: In Homam IV was Quote like: "Rufux will crush this army like a bug." If has scounting and has bigger army. So "like a bug" is little easter egg referencing on this.
+
+\subsection*{\MakeUppercase{Scenario Length}}
+
+This Scenario is played over 7 Rounds.
+
+\subsection*{\MakeUppercase{Player Setup}}
+
+\textbf{Player Count:} 1--6
+
+\textbf{Starting Resources:} 14 \svg{gold}, 5 \svg{building_materials}, 2 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 2 \svg{building_materials}, 0 \svg{valuables}
+
+\textbf{Starting Units:}
+\begin{itemize}
+  \item  1 × A Few \svgunit{bronze} Units of your choice
+\end{itemize}
+
+\textbf{Town Buildings:} \svgunit{bronze} Dwelling.
+
+\textbf{Map Tile Pool:} Take $P$ × random \mbox{Far (II--III)} Map Tiles to form a shared pool, where $P$ equals the number of players. None of these Tiles may contain a Settlement.
+
+\subsection*{\MakeUppercase{Map Setup}}
+
+Take the following Map Tiles and arrange them as shown in the Scenario map layout.
+Place Far (II--III) Map Tiles with Settlements that match your Faction's Towns. Mark these with a colored Cube (see example).
+
+\begin{itemize}
+  \item P × Starting (I) Map Tile
+  \item P × Far (II--III) Map Tile with the Settlement Field of the same Faction as player's Town
+  \item 2P × Far (II--III) Map Tile without the Settlement Field
+\end{itemize}
```

Comment:
```suggestion
Take the following Map Tiles and arrange them as shown in the Scenario map layout:
\begin{itemize}
  \item P × Starting (I) Map Tile
  \item P × Far (II--III) Map Tile of your Town's home terrain that contains a Settlement
  \item 2P × Far (II--III) Map Tile without a Settlement
\end{itemize}
Place each Far (II--III) Map Tile containing a Settlement adjacent to the Starting (I) Map Tile of the same terrain type. Mark these Settlement tiles with a colored Cube (see example).
```
In the video game, Settlements belong to a faction, but in the adaptation this distinction has little sense. What these tiles have in common is terrain type. Also, current wording above the itemize list doesn't really explain what needs to be done. It is explained visually on the picture only.

Reply:
"Mark these Settlement tiles with a colored Cube (see example)."

This is not supposed to be marked by cube a tiles, but tiles marked by cube on map-template to know which far tiles with settlemens are where.

I think each owner knows wihch settlement is which faciton :D..  and it can be found in rules too (components of each faction),  If somebody do mistake with this, its not a deal for disbalance of scenario.  So i would keep that Faction wording. Cuz its lead to no question. Terrain wording leads to question about castle vs rampart settlements.

Reply:
Ah, right, so this cube thing could be explained differently. I seriously thought that you're supposed to put them on the undiscovered tiles. Inferno campaign 3rd scenario does this very well: https://github.com/qwrtln/Homm3BG-mission-book/blob/main/assets/maps/inferno_steadwicks_fall.png

Also, I disagree with the faction thing making sense, as it makes sense only to people who played the video game, and can recognize those settlements visually. Many people asked on BGG and Discrod about Tower's settlement (not shown in the rule book), because they had no idea what it was. But it's your call, we can leave it at that.

Reply:
Im thinking, that in this version of scenario is not needed that rule... Only if they will use optional rule for same terrian. 

But its not matter, what settlment is where (in that case that you dont need conquer your faction settlement)

Reply:
So reworked it, this is outdated, pls check new version and create new discussion?